File: holbertonschool-higher_level_programming/discover_ruby/github_most_popular.rb
require 'HTTPClient'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 1b97b1e74f551c675ad4bd92b6c327bd0fe2253d'
}

uri = 'https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc'

clnt = HTTPClient.new
puts clnt.get_content(uri, extheaders)

File: holbertonschool-higher_level_programming/discover_more_python/github_most_popular.py
import json
import urllib
import urllib2

head = {
    'User-Agent': 'Holberton_school',
    'Authorization': 'token 1b97b1e74f551c675ad4bd92b6c327bd0fe2253d'
}

url = 'https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc'

req = urllib2.Request(url, headers=head)
res = urllib2.urlopen(req)
ans = res.read()

print(ans)
